Gemfile

```diff
@@ -13,6 +13,7 @@ gem "rubocop", "~> 0.54.0", require: false
 gem "sass-rails", "~> 5.0"
 gem "turbolinks", "~> 5"
 gem "uglifier", ">= 1.3.0"
+gem "hirb"
```

Comment:
what is this ?

Reply:
em định dùng gem này để hiển thị bảng csdl trong rails c cho dễ nhìn, nhưng k dùng đc